### ISSUE TITLE
[android] Missing accessibility content labelling

### ISSUE THREAD

**Issue by rtsisyk** [organicmaps/organicmaps#6638]
Content labelling: 17 unique issues (14 warnings, 3 minor issues)

https://support.google.com/accessibility/android/answer/7158690?hl=en

<img width="839" alt="image" src="https://github.com/organicmaps/organicmaps/assets/1799054/9fce708b-26d7-44b5-9add-0a92782e2b81">
<img width="846" alt="image" src="https://github.com/organicmaps/organicmaps/assets/1799054/2032f11d-d45d-4467-a8bc-154ec66f3456">
<img width="846" alt="image" src="https://github.com/organicmaps/organicmaps/assets/1799054/22c5d078-972f-42a4-aa05-a22c2353d27d">
<img width="846" alt="image" src="https://github.com/organicmaps/organicmaps/assets/1799054/e23206ea-5b8f-4ec1-bf3e-05b6cb761a21">
<img width="846" alt="image" src="https://github.com/organicmaps/organicmaps/assets/1799054/74bc2a8f-cd06-43f8-8b63-c31d7866d09f">
<img width="818" alt="image" src="https://github.com/organicmaps/organicmaps/assets/1799054/e4a86a73-766f-485f-8212-a994c65db581">

Run `./gradlew lint` to get the full report.

**Comment by ashutosh-kumar-kushwaha** [comment]
I want to work on this. Please assign this to me.

**Comment by Jean-BaptisteC** [comment]
Feel free PR, don't ask to work on issues

**Comment by SigmaZardan** [comment]
@ashutosh-kumar-kushwaha  Are you still working on this one ?
